recipes/meme/meta.yaml

```diff
@@ -13,6 +13,8 @@ source:
 build:
   number: 0
   detect_binary_files_with_prefix: True
+  run_exports:
+    - {{ pin_subpackage("meme", max_pin="x.x") }}
```

Comment:
Please add a comment with a rationale for x.x pinning instead of just x here. E.g. that it is known that the package breaks frequently API or CLI in the minor version

Reply:
I have misunderstood max_pin and changed 'x.x' to 'x' in my next commit.